<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SARAGARW-LAP" ignored="0" name="Test" tests="3" failures="1" timestamp="02 Feb 2018 06:41:26 GMT" time="29.773" errors="0">
  <testcase name="testcase123" time="23.361" classname="com.TestCases.Test123">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Sucess msg not Displayed...">
      <![CDATA[java.lang.AssertionError: The following asserts failed:Sucess msg not Displayed...at com.library.library.verifysucessmsg(library.java:55)at com.TestCases.Test123.testcase123(Test123.java:21)... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testcase123 -->
  <testcase name="testcase123" time="2.806" classname="com.TestCases.Test123"/>
  <testcase name="testcase123" time="2.921" classname="com.TestCases.Test123"/>
</testsuite> <!-- Test -->
